composer.json

```diff
@@ -19,15 +19,15 @@
   ],
   "license": "Apache-2.0",
   "require": {
-    "php": "8.0.* | 8.1.* | 8.2.*",
+    "php": "8.1.* || 8.2.*",
     "illuminate/support": "^9.0 || ^10.0",
```

Comment:
and can we please also drop illuminate 9.0 support and add 11.0 (for all illuminate packages)